presidio-analyzer/presidio_analyzer/presidio_logger.py

```diff
@@ -5,24 +5,27 @@
 
 class PresidioLogger:
     """A wrapper class for logger"""
-    def __init__(self, logger_name="presidio", default_level="INFO"):
+
+    def __init__(self, logger_name: str = "presidio", default_level: str = "INFO"):
         if logger_name:
             logger = logging.getLogger(logger_name)
+            logger.addHandler(logging.NullHandler())
```

Comment:
default logger has a `NullHandler` per https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library